aten/src/ATen/native/cuda/SpectralOps.cpp

```diff
@@ -234,6 +234,22 @@ void cufft_clear_plan_cache_impl(int64_t device_index) {
 namespace {
 constexpr int64_t cufft_max_ndim = 3;
 
+// "Large" here means a prime factor not special-cased by cuFFT
+bool has_large_prime_factor(int64_t n) {
```

Comment:
Elaborate in this comment on the intended use of this function and what constitutes a "large" prime factor (and why that's "large" in this context)